01_html_basics/starter_code/cookie_recipe/index.html

```diff
@@ -14,27 +14,75 @@
 - The color for the text reading "Nutrition Information" is #887
 - The link color is #900
 
-Bonus: The border on the page and around the image is black
+Bonus: The border on the page and around the image is black -->
 
-The Cookie Recipe
-=================
+<!-- The Cookie Recipe -->
+
+<!-- ================= -->
+
+
+<h1>The Best Chocolate Chip Cookies</h1>
+
+<br>
```

Comment:
Also not incorrect technically, but your use of the `<br>` tag throughout the document is something I would advise against. The reason is subtle, but not unfounded. Basically it's that we want to keep our markup as purely structural as possible, and with as little influence on _style_ as possible, and leave all presentational decisions to our CSS, because that's where we'll be doing all of our styling anyhow, right? So let's say you hop into this project and you need to adjust some styles, including the spacing between the elements — naturally you're going to crack open the CSS document to do it, but you'll get there and find that the space between things isn't even accounted for there...hmmm, then you realize, oh, that space is hard-coded as `<br>` tags throughout the document! Now I've gotta remove all of that markup! So basically, the way in which you used the break tags is for presentational purposed, because you wanted to space them out, which should be left to the CSS to handle. Now, I realize at this point we hadn't introduce CSS, so it's possible you wanted to space 'em out really badly and used whatever means you could, even though you understand you'd ordinarily handle that in CSS, in which case, ignore my long-winded response :)